projects/dotcms-field-elements/src/components/dot-form/dot-form-column/dot-form-column.e2e.ts

```diff
@@ -0,0 +1,125 @@
+import { newE2EPage, E2EPage, E2EElement } from '@stencil/core/testing';
+import { DotCMSContentTypeField, DotCMSContentTypeRow } from './../models';
+
+const basicField: DotCMSContentTypeField = {
+    clazz: '',
+    contentTypeId: '',
+    dataType: '',
+    defaultValue: '',
+    fieldContentTypeProperties: [],
+    fieldType: '',
+    fieldTypeLabel: '',
+    fieldVariables: [],
+    fixed: true,
+    hint: '',
+    iDate: 100,
+    id: '',
+    indexed: true,
+    listed: true,
+    modDate: 100,
+    name: '',
+    readOnly: true,
+    regexCheck: '',
+    required: true,
+    searchable: true,
+    sortOrder: 100,
+    unique: true,
+    values: '',
+    variable: ''
+};
+
+const fieldsMock: DotCMSContentTypeRow[] = [
+    {
+        columns: [
+            {
+                fields: [
+                    {
+                        ...basicField,
+                        variable: 'textfield1',
+                        required: true,
+                        name: 'TexField',
+                        fieldType: 'Text'
+                    },
+                    {
+                        ...basicField,
+                        defaultValue: 'key|value,llave|valor',
+                        fieldType: 'Key-Value',
+                        name: 'Key Value:',
+                        required: false,
+                        variable: 'keyvalue2'
+                    }
+                ]
+            }
+        ],
+    }];
+
+describe('dot-form-column', () => {
+    let page: E2EPage;
+    let element: E2EElement;
+
+    beforeEach(async () => {
+        page = await newE2EPage({
+            html: `<dot-form></dot-form>`
```

Comment:
```suggestion
            html: `<dot-form-column></dot-form-column>`
```

Reply:
This is not the right approach, you don't need to test how the `dot-form-column` works inside the `dot-form` you need to test that the `dot-form-column` works as expected.